.github/CODEOWNERS

```diff
@@ -492,10 +492,10 @@
 # ServiceLabel: %Defender EASM
 # ServiceOwners:                                                   @thang-bit
 
-# PRLabel: %Durable Task
+# PRLabel: %Durable Task Scheduler
```

Comment:
This is not a change that you can make arbitrarily.   You'll need to work with @ronniegeraghty for an Azure SDK label name.